### PR TITLE
Measuring success fixes and minor content changes

### PR DESCRIPTION
**Pull request by keelanfh**
Some fixes and minor content changes in the **Measuring success** section, after some work by Laura looking at this. Includes fixes for these issues:

* Using the Google Analytics practice account - remove practice account details and rename page (perhaps ‘Google Analytics Training’)
* Can we add a note to say these training resources relate to Universal Analytics only and direct them to Digital.Insights@Essex.gov.uk if they’re looking for training on Google Analytics 4?
* ECC Web Analytics Framework page – still features Chris’ email. Can we update to Digital.Insights@Essex.gov.uk?
* ECC Web Analytics Framework page – can’t read what’s in the diagram.. can we make it bigger? 
* Finding your way around – ‘next’ link is broken (goes through to 404 page)
* A couple of other things

I've tried the changes out on another Github pages instance for testing, and looks OK to me, but would be great if someone else could look it over, and merge in when happy! Any questions, let me know.

### CODE REVIEW

File: Measuring-success/Using-data-to-improve-your-service.markdown [hunk 1]
```diff
@@ -35,117 +35,9 @@ Think about:
...
-      <tr>
-        <td>Number of returning visits compared to new visits </td>
-        <td>Segmented</td>
-      </tr>
-    </tbody>
-</table>
-</div>
+A web measurement forms the basis of reports and dashboards and will change slightly from service to service. 
+
+[See an example of a web measurement framework we use at Essex County Council (ECC) for measuring performance](https://keelanfh.github.io/essex-county-council-digital-manual/Measuring-success/Google-analytics/Ecc-web-analytics-framework).
```
Comment (tomouchuu) [inline Measuring-success/Using-data-to-improve-your-service.markdown]: This looks to be the only instance of it but worth checking to confirm the links are relative or direct to the digital manual

**Review by tomouchuu — CHANGES_REQUESTED** [review]

File: Measuring-success/Using-data-to-improve-your-service.markdown (hunk 1 above)
Comment (keelanfh) [reply Measuring-success/Using-data-to-improve-your-service.markdown]: Oops, great spot, thanks! Should be fixed now. 🤭 I can't spot any other instances either.

**Review by tomouchuu — APPROVED** [review]
Looks all good to me now. If you're happy I can get this merged

**Comment by keelanfh** [comment]
> Looks all good to me now. If you're happy I can get this merged

Yep, happy with this, thank you!

**Comment by tomouchuu** [comment]
Changes are now live @keelanfh

**Comment by keelanfh** [comment]
> Changes are now live @keelanfh

🎉🎉🎉 Thank you!

**Pull request merged**